src/govuk/components/header/template.test.js

```diff
@@ -136,6 +136,15 @@ describe('header', () => {
       expect($nav.attr('aria-label')).toEqual('Menu')
     })
 
+    it('renders navigation label correctly when custom menu button text is set', () => {
```

Comment:
Do we have a test for when we're setting `menuButtonText` and `navigationLabel`?

Reply:
We do now !